Changes WMS layer to the Retina layer on MapProxy.

BobTorgerson opened the pull request:
This PR is to remove the dependency of the old MapProxy layer alaska_osm that is being removed. We instead want to make use of the alaska_osm_retina layer that has retina tiles.

To use this branch, be sure to use NVM to switch to v10.24.0 before installing and running.

nvm install 10.24.0
npm install
npm run dev

Service runs on http://localhost:8080

Ensure the map base layers are showing up for this change to be merge worthy.

brucecrevensten reviewed (commented):
<img width="261" alt="Screen Shot 2023-03-22 at 4 53 13 PM" src="https://user-images.githubusercontent.com/525049/227064342-f62ab6b6-5663-4c35-8c91-f352f203e917.png">

We kinda need a no-labels version of the retina layer in order for this to work.  Let's discuss.

BobTorgerson commented:
@brucecrevensten I've added the newly created retina layer that has no place names on the map that I've replaced the base map with. It's available through Geoserver as alaska_osm_retina_no_places

BobTorgerson commented:
@brucecrevensten This is good to go when you want to have a look, it will allow us to finish the Atlas21 MapProxy removal ticket: https://trello.com/c/wjonVTjx/75-deprecate-remove-all-infra-data-layers-for-old-osm-tiles-in-geoserver-atlas21-etc
